src/views/transactions/components/transaction-form/transaction-form.view.jsx

```diff
@@ -69,7 +70,9 @@ function TransactionForm({
 
   React.useEffect(() => {
     if (estimatedDepositFeeTask.status === "successful") {
-      setGasPrice(estimatedDepositFeeTask.data.gasPrice);
+      if (estimatedDepositFeeTask.data.gasPrice !== null) {
```

Comment:
After changing the interface we can also remove this check I think.

Reply:
Right! Done.